lib/notifier.js

```diff
@@ -25,18 +27,29 @@ function notifyIfChanged(businesses) {
 }
 
 function notifyConsole(message, options){
+    let date_ob = new Date();
     if(options.clear){
         console.clear();
     }
+    if(options.include_time) {
```

Comment:
This feature might make sense, but it would require a separate pull request. Never combine different features in a single pull request, unless you discussed it with the author in advance.